honeybadger/config.py

```diff
@@ -59,3 +61,13 @@ def is_dev(self):
         :rtype: bool
         """
         return self.environment in self.DEVELOPMENT_ENVIRONMENTS
+
+    @property
+    def is_aws_lambda_environment(self):
+        """
+        Checks if you are in an aws lambda environment by checking for the existence
+        of "AWS_EXECUTION_ENV" in the environment variables.
+
+        :rtype: bool
+        """
+        return os.environ.get("AWS_EXECUTION_ENV") is not  None
```

Comment:
According to [this PR](https://github.com/honeybadger-io/honeybadger-ruby/pull/384), `AWS_LAMBDA_FUNCTION_NAME` is a more accurate variable for detecting the Lambda environment.

```suggestion
        return os.environ.get("AWS_LAMBDA_FUNCTION_NAME") is not  None
```